src/equations/compressible_euler_potential_temperature_1d.jl

```diff
@@ -36,6 +34,48 @@ end
 varnames(::typeof(cons2prim),
 ::CompressibleEulerPotentialTemperatureEquations1D) = ("rho", "v1", "p1")
 
+# Calculate 1D flux for a single point
+@inline function flux(u, orientation::Integer,
+                      equations::CompressibleEulerPotentialTemperatureEquations1D)
+    rho, rho_v1, rho_theta = u
+    v1 = rho_v1 / rho
+    p = equations.K * exp(log(rho_theta^equations.gamma))
```

Comment:
```suggestion
    p = equations.K * rho_theta^equations.gamma
```

`exp(log(x))` should be `x`? Or do you mean

```suggestion
    p = equations.K * exp(equations.gamma * log(rho_theta))
```

?